src/components/activity-header/header.tsx

```diff
@@ -27,7 +28,9 @@ export class Header extends React.PureComponent<IProps> {
       <div className="activity-header" data-cy="activity-header">
         <div className={`inner ${fullWidth ? "full" : ""}`}>
           <div className={`header-left ${linkClass}`} onClick={this.handleProjectLogoClick(projectURL)}>
-            {logo && <img src={logo} className="project-logo" data-cy="project-logo" />}
+            {logo && <img src={logo} className="project-logo" alt="Project website" data-cy="project-logo" 
+                      onClick={this.handleProjectLogoClick(projectURL)} onKeyDown={this.handleProjectLogoClick(projectURL)}
+                      tabIndex={0}/>}
```

Comment:
Might be easier to read these long prop lists that flow onto multiple lines by putting each prop on a line:
```
            {logo && <img 
                        src={logo} 
                        className="project-logo" 
                        alt="Project website" 
                        data-cy="project-logo" 
                        onClick={this.handleProjectLogoClick(projectURL)} 
                        onKeyDown={this.handleProjectLogoClick(projectURL)}
                        tabIndex={0}
                      />}
```